src/_utils/item/Item.tsx

```diff
@@ -134,7 +137,7 @@ const Item = (props: ItemProps) => {
           <Text
             className="kirk-item-body"
             display={leftBodyDisplay}
-            textColor={leftBodyColor}
+            textColor={disabled ? color.fadedText : leftBodyColor}
```

Comment:
You can override `Text` to avoid repetion
```suggestion
const StyledText = styled(Text)`
   color: ${props => (props.isDisabled ? color.disabled : '')};
`
```

Reply:
Overriding `Text` this way did not work since I do need to take pass a variable color in case it's not disabled.

But still, I factorized the change in a separate `getTextColor` method to avoid repetition